bundle/metadata/annotations.yaml

```diff
@@ -4,8 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: nvidia-network-operator
-  operators.operatorframework.io.bundle.channels.v1: stable,v25.1
-  operators.operatorframework.io.bundle.channel.default.v1: v25.1
+  operators.operatorframework.io.bundle.channels.v1: v25.1.0,stable
```

Comment:
channels should be without minor

Reply:
Did you run make bundle?
I don't see the doca-driver udpated

Reply:
When do we update Chart.yaml ?

Reply:
fixed

Reply:
Chart.yaml is already updated